content/applications/websites/ecommerce/checkout_payment_shipping.rst

```diff
@@ -0,0 +1,10 @@
+:nosearch:
+
+===============================
+Checkout, payment, and shipping
+===============================
```

Comment:
Isn't it a bit overkill to have a toctree page for a single sub-page?

Reply:
It's part of a bigger eCommerce doc overhaul https://docs.google.com/spreadsheets/d/1O7Jg4T8kPE0HaLw-3YDjaj54MMk957-9akBFaSNKQA0/edit#gid=0, it'll all be merged into one mega PR and this one will be closed then

Reply:
Ok then, but I strongly advise *against* merging tons of changes at once with a single "mega PR". This has been tried over and over again, and it was always a bad idea :sweat_smile: